### ISSUE TITLE
Suggestion: Replace DuckSearchAgent with WikiSearchAgent in example/SearchAgent.py

### ISSUE THREAD

**Issue by VigneshHexo** [SalesforceAIResearch/AgentLite#25]
Everytime I pull the recent updates from Agentlite master some or the other problem happens in example/SearchAgent.py because of DuckSearchAgent. I think it is better to make WikiSearchAgent as the default Agent as it is much more stable. Please let me know your thoughts on the same

**Comment by JimSalesforce** [comment]
Yes, that's a good suggestion. i also find this problem. we will update it to be WikiSearchAgent. And tools are also already replaced from langchain tools to original packages.
